.gitignore

```diff
@@ -20,3 +20,4 @@ test
 .nyc_output
 doc/*
 !doc/gltf.png
+.github/*
```

Comment:
Just wondering why you're deleting/ignoring these as it looks like they're generated by `propeller-system` https://github.com/PropellerAero/gltf-pipeline/blob/98a7fc65f1852e91141ae155338fbca98149aec3/.github/dependabot.yml

Reply:
oh, thats odd

Reply:
I think you missed the associated file deletes:
![image](https://user-images.githubusercontent.com/484870/187608581-220ff9fa-e577-4ec3-82ea-fb5e74931b21.png)